go/service/main.go

```diff
@@ -325,7 +335,7 @@ func (d *Service) startupGregor() {
 		// Create gregorHandler instance first so any handlers can connect
 		// to it before we actually connect to gregor (either gregor is down
 		// or we aren't logged in)
-		d.gregor = newGregorHandler(d.G())
+		d.gregor = newGregorHandler(globals.NewContext(d.G(), d.ChatG()))
```

Comment:
`ChatG` just returns `globals.ChatContext`, not `globals.Context` (that might be too confusing). `globals.ChatContext` does not have `libkb.G` stuff on it, and `gregorHandler` needs that.